llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -77,7 +77,7 @@ def __init__(
             emb_pdrop (float): The dropout probability for the embedding layer.
             learned_pos_emb (bool): Whether to use learned positional embeddings
             attn_config (Dict): A dictionary used to configure the model's attention module:
-                attn_type (str): type of attention to use. Options: multihead_attention, multiquery_attention
+                attn_type (str): type of attention to use. Options: multihead_attention, multiquery_attention, grouped_query_attention
```

Comment:
add something like: `kv_n_heads (int): number of heads in grouped_query_attention if using grouped_query_attention`